packages/markdown/loader.js

```diff
@@ -198,6 +198,7 @@ module.exports = async function demoLoader() {
         }
       }
     }
+    return relativeModuleFilename;
```

Comment:
I decided to update the method to return the file as it is needed when iterating the relative modules.